changelog.md

```diff
@@ -1,4 +1,8 @@
 # Change Log
+## 4.4.5
```

Comment:
Should add as a feature under 4.4.4.

Reply:
ah sorry, fixing it

Reply:
See that you fixed the changelog, thanks